tf-modules/cumulus/ecs_cluster.tf

```diff
@@ -192,12 +192,9 @@ resource "aws_security_group" "ecs_cluster_instance" {
   tags   = var.tags
 }
 
-resource "aws_security_group_rule" "ecs_cluster_instance_allow_ssh" {
-  type              = "ingress"
-  from_port         = 2
-  to_port           = 2
-  protocol          = "-1"
-  cidr_blocks       = ["0.0.0.0/0"]
+resource "aws_vpc_security_group_ingress_rule" "ecs_cluster_instance_allow_ssh" {
+  ip_protocol       = "-1"
+  cidr_ipv4         = cidrsubnet(data.aws_vpc.current.cidr_block, 4, 1)
```

Comment:
This appears to be the recommended approach. Open to pushback.

Reply:
https://developer.hashicorp.com/terraform/language/functions/cidrsubnet

Reply:
Also https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/vpc

Reply:
@npauzenga  what's the reason for `4, 1`?     Won't that result in a.b.c+1.d/24, which is far more restrictive, and excludes VPC subnets?

Reply:
@npauzenga  also.... I'm curious why we need this rule *at all*.

Reply:
Per Slack: we probably don't. The latest couple of commits remove this rule. See full changeset as it's two files and easier to grok than looking at individual commits 😄